gt/main.c

```diff
@@ -550,18 +673,52 @@ gt_proc(void *arg)
 				continue;
 			}
 
+			/*
+			 * If it is a fragmented packet,
+			 * then try to reassemble.
+			 */
+			if (pkt_info.frag == true) {
+				complete_pkt = gt_reassemble_incoming_pkt(
+					m, cur_tsc, &pkt_info, instance);
```

Comment:
Do we need `complete_pkt`? Or can we just set the return value of this function to `m`? Since the old `m` is never used again.